db/mockDGen.js

```diff
@@ -1,10 +1,46 @@
+//generates and modifies data:
+const fs = require('fs');
+const mapData2 = require('./data/mapData2');
```

Comment:
Maybe some more descriptive variable names here, so I know which data you're working with? What's the difference between MapData1 and MapData2 so I don't have to go into those files to look.

Reply:
Yeah this file is something I didn't mean to commit. I basically had to modify the data I had generated earlier because Kevin is relying on the address data im using in his implementation. I need to write data to a file and keep it in my file because of this.